internal/util/mqclient/pulsar_consumer.go

```diff
@@ -104,8 +105,16 @@ func (pc *PulsarConsumer) Ack(message Message) {
 
 // Close the consumer and stop the broker to push more messages
 func (pc *PulsarConsumer) Close() {
-	pc.c.Close()
-	close(pc.closeCh)
+	if !pc.isClosed {
+		// Unsubscribe for the consumer
+		err := pc.c.Unsubscribe()
+		if err != nil {
+			panic(err)
```

Comment:
Should the process panic if the `Unsubscribe` failed? I dont think the failure of `Unsubscribe` is severe like that.

Reply:
panic and would retried by Coord